### ISSUE TITLE
Units not Consistent Throughout 

### ISSUE THREAD

**Issue by Aeroengineer1** [GoldenCheetah/GoldenCheetah#4069]
I am using the latest August 3.6 release on Win 10 - 64 bit Pro.  On the overview page, even though I have set the units to US customary/Imperial units it is still showing my weight in kg on the overview tab.  In one it is labeled as kg and the other it is labeled in lbs.

![image](https://user-images.githubusercontent.com/5205655/141693679-926c67f0-589f-4c1d-88b1-cd622ef201dd.png)

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by amtriathlon** [comment]
> I am using the latest August 3.6 release on Win 10 - 64 bit Pro.

This is an outdated dev build, please see releases or the main page for the last one and recreate the chart after installing the new version.

**Comment by Aeroengineer1** [comment]
Sorry for filing issues, I can see that this community is not particularly open. I will go back to the released version and not worry about filing issues.

**Comment by liversedge** [comment]
We have a forum for user support please use that.
The github issues are for reporting bugs and feature requests.
I would also encourage you to read the FAQ.
